src/main/java/org/ditang/relaxng/defaults/RelaxDefaultsParserConfiguration.java

```diff
@@ -149,5 +158,23 @@ protected void insertRelaxDefaultsComponent() {
         next.setDocumentSource(fRelaxDefaults);
     }
   }
+  /**
+   * @see org.apache.xerces.parsers.XIncludeAwareParserConfiguration#setFeature(java.lang.String, boolean)
+   */
+  @Override
+  public void setFeature(String featureId, boolean state) throws XMLConfigurationException {
+    if(XERCES_SCHEMA_VALIDATION.equals(featureId)) {
+      relaxNGValidation = state;
+    } else {
+      super.setFeature(featureId, state);
+    }
+  }
   
+  @Override
+	public void setProperty(String propertyId, Object value) throws XMLConfigurationException {
```

Comment:
The indentation in this PR uses both 2 and 4 spaces. Could you use 4 spaces for all indents?